resource-validator/src/main/scala/com/broadinstitute/dsp/resourceValidator/StoppedRuntimeChecker.scala

```diff
@@ -43,7 +44,7 @@ object StoppedRuntimeChecker {
           runtimeOpt <- deps.computeService
             .getInstance(runtime.googleProject, runtime.zone, InstanceName(runtime.runtimeName))
           runningRuntimeOpt <- runtimeOpt.flatTraverse { rt =>
-            if (rt.getStatus == Instance.Status.RUNNING)
+            if (rt.getStatus.toUpperCase == Instance.Status.RUNNING.name().toUpperCase)
```

Comment:
How do we handle other statuses for GCE or Dataproc? I know Galaxy is the only resource we do all uppercase for in terms status - do we do `.toUpperCase` everywhere to be consistent?

Reply:
the upper case here is just for this particular comparison...we could use `toLowerCase` here too..the point is to make sure casing is consistent for both left and right side

> How do we handle other statuses for GCE or Dataproc? 
This check is to make sure all `Stopped` should be indeed `Stopped` in GCP. I guess I can make the check to error if the VM status is not `Stopped` or `Terminated` to make it a bit more inclusive